.gitignore

```diff
@@ -1,3 +1,5 @@
 lib
 node_modules
 coverage
+npm-debug.log
+examples/basic/npm-debug/.log
```

Comment:
I think there's a typo in the last one? And I don't think it's needed when you already have `npm-debug.log` there.